hyperspy/tests/io/test_bcf.py

```diff
@@ -112,7 +112,7 @@ def test_hyperspy_wrap():
                                                                       'azimuth_angle': 90.0}},
                                                  'stage_x': 62409.2,
                                                  'magnification': 131.1433,
-                                                 'tilt_stage': 0.0}},
```

Comment:
this was mistake and missconseption that bruker does not read the stage tilt value from microscope;
0.0 is tilt angle available to setup in separate Esprit window, stage tilt is actually 0.5 and is read from microscope.
P.S. this is horizontal position of German precision stage LOL